portals/knowledge-management/src/Components/Footer.vue

```diff
@@ -50,10 +50,11 @@
 
         <div class="flex w-full justify-center pb-4 pt-2">
             <span class="w-11/12 text-center text-xs lg:w-3/4 xl:w-7/12">
-                © 2016-{{ new Date().getFullYear() }} Canyon GBS LLC. All Rights Reserved. Canyon GBS™ and Aiding
-                App™ are trademarks of Canyon GBS LLC. For more information or inquiries, please visit our website at
+                © 2016-{{ new Date().getFullYear() }} Canyon GBS LLC. All Rights Reserved. Canyon GBS&reg; and Aiding
+                App&reg; are trademarks of Canyon GBS LLC. For more information or inquiries, please visit our website
+                at
```

Comment:
Though `&reg;` is typically fine in almost all cases, lets use the numeric version `&#174;` as it has the most compatibility between the two.

Reply:
Done